pkg/ddl/executor.go

```diff
@@ -7083,16 +7091,16 @@ func (e *executor) RefreshMeta(sctx sessionctx.Context, args *model.RefreshMetaA
 		Version:        model.JobVersion2,
 		SchemaID:       args.SchemaID,
 		TableID:        args.TableID,
+		SchemaName:     args.SchemaName,
+		TableName:      args.TableName,
```

Comment:
don't set it, `TableName` is for real table, InvolveAll is not a valid value for this field